x-pack/plugins/security_solution/server/lib/detection_engine/signals/utils.ts

```diff
@@ -750,6 +756,7 @@ export const mergeReturns = (
       success: existingSuccess && newSuccess,
       warning: existingWarning || newWarning,
       searchAfterTimes: [...existingSearchAfterTimes, ...newSearchAfterTimes],
+      enrichmentTimes: [...existingEnrichmentTimes, ...newEnrichmentTimes],
```

Comment:
As opposed to `combineConcurrentResults`, `mergeReturns` concatenates the durations.